examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
Yeah this is why I'd suggested just adding the probe inline. If you move to test/ you'll need to make sure the directory you put it in is included in the test tarball. @ixdy where would be a safe place to put yaml? IIRC the last time I shoved something in test/e2e/testing-manifests it wasn't available on the server, and I ended up just inling it anyway